src/EmployerAan/SFA.DAS.EmployerAan/Models/ApprenticeshipDetail.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.EmployerAan.Models;
+
+public class Apprenticeship
+{
+    public int ActiveApprenticesCount { get; set; }
+    public IEnumerable<string> Sectors { get; set; } = Enumerable.Empty<string>();
```

Comment:
Sectors exist in here as well as in ApprenticeshipsFilterValues, is that correct?

Reply:
Yes